src/test/converter/nested-modules/nested-modules.ts

```diff
@@ -0,0 +1,45 @@
+module TestModule
```

Comment:
:+1: for adding a test case. Doesn't TypeScript recommend using the `namespace` keyword now? My understanding was that those two keywords were aliases for each other.

Reply:
Some existing test cases use the "module" keyword. And no linter argued ;) So...

According to [TypeScript](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html) it is distinguished between internal and external modules.

So I would say if module system is "none", everything should be "namespace", although it makes no real difference in output.